packages/contracts-bedrock/test/L1/OptimismPortal.t.sol

```diff
@@ -830,6 +831,7 @@ contract OptimismPortal_FinalizeWithdrawal_Test is CommonTest {
 
         // Mock a call where the resulting output root is anything but the original output root. In
         // this case we just use bytes32(uint256(1)).
+        // nosemgrep: sol-style-use-abi-encodecall
         vm.mockCall(
             address(optimismPortal.l2Oracle()),
             abi.encodeWithSelector(IL2OutputOracle.getL2Output.selector),
```

Comment:
Not really since abi.encodeCall requires inputs to be given but encodeWithSelector does not, the intention of the test was to mock calls to the function with any input

Reply:
Hmmm ok I think we should use `abi.encodePacked` instead. `abi.encodeWithSelector` is weird usage here, I know it's written like this in the foundry docs but I don't like *deliberately* encoding something improperly when what we're actually doing is just providing the 4byte selector.

Reply:
I think something like this should work `abi.encodeCall(IL2OutputOracle.getL2Output, (0)),` given that `mockCall` supports loose matching...